src/Articles/tests/containers/CreateArticle.test.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { CreateArticle, mapStateToProps } from '../../containers/CreateArticle';
+
+let mockedState = {
+  isFieldInvalid: {
+    title: false,
+    body: false,
+    tags: false,
+  },
+  articleTitle: 'hello',
+  articleBody: 'body',
+  articleTags: 'body, boys',
+  articlePhoto: {
+    name: 'No Photo selected',
+    data: null,
+  },
+};
+
+
+// test
```

Comment:
PLease remove this l think its unecessary

Reply:
@jennizalwango noted.